### PR TITLE
Document GraphQL rate limiting removal

### PR DESCRIPTION
**Pull request by real34**
This PR documents the feature removal from https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/2624

## Preview

- [Features removal](https://deploy-preview-787--heuristic-almeida-1a1f35.netlify.app/docs/remixed/migration/features-removal) (new page)
- [Deprecated code - new section](https://deploy-preview-787--heuristic-almeida-1a1f35.netlify.app/docs/remixed/migration/deprecated-code-removal#features)

### CODE REVIEW

**Comment by real34** [comment]
Build broken because of https://github.com/front-commerce/developers.front-commerce.com/pull/786, you can ignore it and review the diff instead … or start approving the other one :angel:

File: docs/migration/deprecated-code-removal.mdx [hunk 1]
```diff
@@ -203,3 +203,13 @@ project.
 - [Remove deprecated `InMemoryCacheWithDeprecation`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/1892)
 - [Remove deprecated `decorateLoggerWithTrace`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/1926)
 - [Remove deprecated express config fields](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/1871)
+
+## Features
+
+Some code has also be removed while not explicitly deprecated in `2.x`. Most of
```
Comment (dpobel) [inline docs/migration/deprecated-code-removal.mdx:209]:
```suggestion
Some code has also been removed while not explicitly deprecated in `2.x`. Most of
```

File: docs/migration/features-removal.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+If you are a Front-Commerce customer using one of this feature in your `2.x`
+project, please <ContactLink>contact us</ContactLink> as you plan your migration
+so we could reevaluate our decision.
+
+:::
+
+## GraphQL Rate Limiting
+
+In version 3.0, we removed the
+[Rate Limiting](http://localhost:3000/docs/2.x/advanced/graphql/rate-limiting)
```
Comment (dpobel) [inline docs/migration/features-removal.mdx:24]:
```suggestion
[Rate Limiting](/docs/2.x/advanced/graphql/rate-limiting)
```

**Review by dpobel — APPROVED** [review]

**Review by Kagaminara — APPROVED** [review]
Nothing more to add to Damien's review 👍 .

**Pull request merged**